client/src/components/questions/Questions.jsx

```diff
@@ -1,44 +1,46 @@
-import React, { useEffect, useContext } from 'react';
-// import styled from 'styled-components';
+import React, { useEffect, useContext, useState } from 'react';
 import parse from '../../parse';
-// import { ProductIDContext } from '../App.jsx';
 import { GlobalContext } from '../GlobalContext.jsx';
 import Search from './Search.jsx';
 import QuestionList from './QuestionList.jsx';
-import AnswerForm from './AnswerForm.jsx';
+import QuestionForm from './QuestionForm.jsx';
 import './questions.css';
 
 function Questions() {
   const { productID } = useContext(GlobalContext);
-  const [questions, setQuestions] = React.useState([]);
-  const [isOpen, setIsOpen] = React.useState(false);
+  const [questions, setQuestions] = useState([]);
+  const [isOpen, setIsOpen] = useState(false);
+  const [allQuestions, setAllQuestions] = useState([]);
 
   useEffect(() => {
-    parse.get(`/qa/questions/?product_id=${productID}&page=30&count=4`)
-      .then((data) => setQuestions(data.results))
+    parse.get(`/qa/questions/?product_id=${productID}&page=1&count=100`)
+      .then((data) => {
+        setAllQuestions(data.results);
+        setQuestions(data.results.slice(0, 4));
+      })
       .catch((err) => console.log(err));
```

Comment:
It might be beneficial to make the error more identifiable. I believe a line such as 
`console.log("Get questions error:", err)`
would be easier to read and see where the error is coming from